src/Reader/Extension/AbstractFeed.php

```diff
@@ -133,9 +133,9 @@ public function setXpath(DOMXPath $xpath = null)
     /**
      * Get the DOMXPath object
      *
-     * @return string
+     * @return DOMXPath
      */
-    public function getXpath()
+    public function getXpath(): DOMXPath
```

Comment:
Please do not add php typehints as this will is a BC break unless the class is final.